paper.md

```diff
@@ -1,36 +1,51 @@
 ---
-title: 'PyGBe: Python, GPUs and Boundary elements for biomolecular electrostatics'
+title: 'PyGBe: Python and GPU Boundary-integral solver for electrostatics'
 tags:
   - electrostatics
   - biophysics
   - Poisson-Boltzmann
+  - nano-plasmonics
+  - lspr
 authors:
- - name: Christopher D. Cooper
-   orcid: 0000-0003-0282-8998
-   affiliation: Universidad Técnica Federico Santa María
  - name: Natalia C. Clementi
    orcid: 0000-0002-0575-5520
    affiliation: The George Washington University
  - name: Gilbert Forsyth
    orcid: 0000-0002-4983-1978
    affiliation: The George Washington University
+ - name: Christopher D. Cooper
+   orcid: 0000-0003-0282-8998
+   affiliation: Universidad Técnica Federico Santa María
  - name: Lorena A. Barba
    orcid: 0000-0001-5812-2711
    affiliation: The George Washington University
-date: 14 July 2016
+date: 12 June 2017
 bibliography: paper.bib
 ---
 
 # Summary
 
-PyGBe—pronounced _pigbē_—is a Python code to apply the boundary element method for molecular-electrostatics 
-calculations in a continuum model.
-It computes solvation energies for proteins modeled with any number of dielectric regions, which is relevant in calculations of protein binding affinity, protein-surface interaction, acid-dissociation contants, among other applications. 
-The mathematical formulation follows @YoonLenhoff1990 for solving the Poisson-Boltzmann equation of the [implicit-solvent](https://en.wikipedia.org/wiki/Implicit_solvation) model in integral form.
+PyGBe—pronounced _pigbē_—is a Python library that uses the boundary integral 
+method applied to biomolecular electrostatics and nanoparticle plasmonics. 
+
+This PyGBe release updates the exisiting library presented in @DCooper2016 to Python 3,
+introduces an application to nanoplasmonics and, it includes better regression tests
+using pytest and a redesign of the convergence tests.
```

Comment:
I have already made edits that I think leave it clear what the main contribution of this release is.